gen/templates/resource_test.go.tmpl

```diff
@@ -497,14 +610,16 @@ func TestAccResource{{$.resourceClassName}}With{{capitalize .class_name}}(t *tes
 			},
 		{{- end}}
 		},
+		CheckDestroy: resource.ComposeAggregateTestCheckFunc(
+			testCheckResourceDestroy,
+		),
```

Comment:
I don't think we need the ComposeAggregateTestCheckFunc() here right? We can directly pass testCheckResourceDestroy no?

Reply:
ComposeAggregateTestCheckFunc() under destroy had two functions in the past. Yes, now we don't need the aggregate function. I replaced it.